src/doc/src/reference/environment-variables.md

```diff
@@ -377,7 +377,8 @@ let out_dir = env::var("OUT_DIR").unwrap();
   not need to run `make -j`, and instead can set the `MAKEFLAGS` env var to the
   content of `CARGO_MAKEFLAGS` to activate the use of Cargo's GNU Make compatible
   [jobserver] for sub-make invocations.
-* `OPT_LEVEL`, `DEBUG` --- values of the corresponding variables for the profile currently being built.
+* `DEBUG` --- `true` or `false`. `false` corresponds to when the debug profile setting is `0`, `false`, or `"none"`.
+* `OPT_LEVEL` --- values of the corresponding variable for the profile currently being built.
```

Comment:
With us splitting these up, we should probably link to the relevant content
- https://doc.rust-lang.org/cargo/reference/profiles.html#debug
- https://doc.rust-lang.org/cargo/reference/profiles.html#opt-level